R/build_stan_data.R

```diff
@@ -67,40 +67,63 @@ sf_none <- function() {
 #' be estimated when sampling.
 #' @inheritParams fit_seromodel
 #' @param group_size Age groups size
-#' @return Integer vector with the indexes numerating each year/age
-#' (depending on the model).
+#' @param model_type Type of the model. Either "age" or "time"
+#' @return Data frame with the indexes numerating each age/year (depending on
```

Comment:
A Data Frame which describes the grouping of years or ages (dependent on model) into pieces within which the FOI is assumed constant when performing model fitting.